app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SessionsController < ApplicationController
+  
+  before_action :user, only: [:create]
+
+  def new
+    if current_user
+      redirect_to user_path(current_user), alert: t(".flash.alert")
+    end
+  end
+
+  def create
+    if user&.authenticate(params[:session][:password])
+      session[:user_id] = user.id
+      redirect_to user_path(user), notice: t(".flash.notice")
+    else
+      flash[:alert] = t(".flash.alert")
+      render :new
+    end
+  end
+
+  def destroy
+    session.clear
+    redirect_to new_session_path, notice: t(".flash.notice")
+  end
+
+  private
+
+  def user
+    @user ||= User.find_by(email: params[:session][:email])
+  end
```

Comment:
Just a note on this method, it was a one liner at the beginning of the create method (modified a bit) and rubocop complained about the length of the method so I moved it out.